levels/sre.tex

```diff
@@ -0,0 +1,368 @@
+
+    \bigbreak
+
+    Is a member of on-call rotation and can resolve issues outlined in
+    runbooks.
+
+    \bigbreak
+
+    Demonstrates knowledge of AWS. May have certification like AWS Certified
+    SysOps Administrator.
+    \\ [13em]
+\hline
+    % Level
+    3
+    &
+
+    % Product
+    \cellcolor{blue!25}
+    Collaborates with the team to scope requirements, based on good
+    understanding of existing longer term product vision and estimates of the
+    system design of a feature of a product.
+    &
+
+    % Communication
+    Supports less experienced peers' technical skills, answering questions and
+    being a resource. Documents and improves team practices.
+    &
+
+    % Networking
+    Can setup and operate a multi-region infrastructure and networking
+    environment. Can reason about it's performance, reliability, and failure modes.
+
+    \bigbreak
+
+    Has in-depth understanding of container networking. Can write own CNI plugin
+    utilizing IPSec or WireGuard on Kubernetes.
+
+    \bigbreak
+
+    Understands advanced networking concepts such as network namespaces, NAT
+    traversal, and BPG.
+    &
+
+    % Security
+    Can build secure systems that will pass quality security audit that will
+    uncover few to no critical system design errors.
+
+    \bigbreak
+
+    Can apply security principals when building systems. Can utilize access
+    control primitives (like IAM and RBAC) to limit access to infrastructure.
+    Understands secret life cycle management in production environments. Understand
+    API authentication systems, can reason about trade-offs between use of JWT,
+    OIDC, and mTLS. Can setup alerting on security systems like CloudWatch.
+
+    \bigbreak
+
+    Understands advanced operating system security concepts. Can utilize
+    Mandatory Access Control (MAC) systems like as SELinux or AppArmor. Understands
+    container security. Can utilize control groups and namespaces to isolate and
+    application.
+    &
+
+    % Systems Engineering
+    &
+
+    % SRE
+    Can use declarative languages like Terraform to build and manage infrastructure.
+
+    \bigbreak
+
+    Demonstrates advanced knowledge of Kubernetes deployments (CRDs, ingress
+    controllers, Load Balancers).
+
+    \bigbreak
+
+    Can build and maintain reliable production CI/CD pipelines for build, test,
+    and release.
+
+    \bigbreak
+
+    Demonstrates advanced knowledge of AWS. May have certification like AWS
+    Certified Solutions Architect.
+    \\ [13em]
+\hline
+    % Level
+    4
+    &
+
+    % Product
+    \cellcolor{blue!25}
+    Leads the implementation of the isolated feature/improvement that
+    measurably and significantly impacts business outcomes from gathering
+    requirements to getting to the market stage.
+    &
+
+    % Communication
+    Writes technical articles/blog posts, delivers tech and lightning talks
+    representing the company's technical vision.
```

Comment:
I would add something along the lines of "Defines or helps to establish SLI/SLO for services and entire products, guides the data collection and setup of the SLI/SLO monitoring". Words are again failing me, but this is an important part of being SRE, and at it gets exponentially harder with scale (both product and team).